src/wrapped_radix_engine_toolkit/high_level/manifest_building/metadata.rs

```diff
@@ -1,15 +1,24 @@
 use crate::prelude::*;
-
 use radix_engine::prelude::ToMetadataEntry as ScryptoToMetadataEntry;
+use strum::*;
 use transaction::prelude::MetadataValue as ScryptoMetadataValue;
 
-#[derive(Debug, PartialEq, Eq, derive_more::Display)]
+#[derive(Debug, PartialEq, Eq, strum::EnumString, strum::Display)]
+#[strum(serialize_all = "snake_case")]
 pub enum MetadataKey {
```

Comment:
The idea here is that we never use string literals for metadata, but rather add them as ""constants"".